src/metrics/include/metrics/v2/event.hpp

```diff
@@ -42,11 +49,22 @@ class OperationEventT;
 
 namespace internals::v2 {
 
+const int NUM_CHANNELS = 4;
+const int BUFFER_SIZE = 1000;
+const int CLIENT_THREADS = 3;
+const int GRPC_THREAD_SLEEP_MS = 50;
+const double SWAP_BUFFER_PERCENT = .25;
```

Comment:
There's not a super motivated reason for these values other than that I ran a lot of patches and these seemed to get the best performance. I wasn't sure if we'd want to make these configurable in the workload yaml; e.g. insert_remove does a bit better with fewer client threads but mixed workloads needs them.